docs/sidebars.js

```diff
@@ -325,6 +325,7 @@ module.exports = {
             "guides/integrate/login-ui/password-reset",
             "guides/integrate/login-ui/logout",
             "guides/integrate/login-ui/oidc-standard",
+            "guides/integrate/login-ui/session-validation",
```

Comment:
I think we should add this at the top, as all the other sites rely on this information.